src/rev_ai/apiclient.py

```diff
@@ -28,51 +28,79 @@ def __init__(self, api_key):
             dashboard on rev.ai
         """
         if not api_key:
-            raise ValueError("API Key cannot be empty.")
+            raise ValueError('api_key must be provided')
+
         self.session = requests.Session()
         self.session.headers.update({
-            'Authorization': 'Bearer {api_key}'.format(api_key=api_key),
+            'Authorization': 'Bearer {}'.format(api_key),
             'User-Agent': 'python_sdk'
         })
 
-    def submit_job_url(self, media_url, options):
+    def submit_job_url(
```

Comment:
Per suggestion and comments from the channel and issue https://github.com/revdotcom/revai-python-sdk/issues/8, I removed the `JobSubmitOptions` and opted for simply adding optional parameters to both of the `submit_job_...` methods.

Reply:
My impression of what was wanted in that discussion is just options=None as the default. Not total removal of the object. I think we should revert this change. What do you think?

Reply:
I think in general, I find it strange to have optional arguments encapsulated into a separate class in Python. I think it's much cleaner and simpler for the user to have each option as an optional parameter of the methods themselves. I might have missed the discussion about this though.

Reply:
I think that makes sense. I know very little about python conventions. If @JDongian agrees I think we can keep it this way